### DISCUSSION TITLE
HmIP-WTH-2 Heizung steuern und Feuchtigkeit anzeigen lassen (beides)

### DISCUSSION THREAD

**Discussion by diegonm1** [thkl/hap-homematic#523] [Q&A]
Hallo zusammen,

vielleicht kann jemand mir helfen. Ich habe ein Thermostat wo man die Heizung steuern kann (mit der Booster Option) und auch die Luftfeuchtigkeit messen.

![image](https://user-images.githubusercontent.com/98041164/150166860-5d841ae4-c612-4239-b93a-40ff519bb809.png)

Leider habe ich entweder ein "RadiatorThermostat" oder "HumidityAcessory". Ich kann nicht beides nutzen. 

![image](https://user-images.githubusercontent.com/98041164/150167150-2c1e977a-8cca-41aa-8d9e-94f6c7de2231.png)

Wenn ich ein Service im Betrieb schon habe, kann das Gerät nicht wieder beifügen.

![image](https://user-images.githubusercontent.com/98041164/150167375-bbc42346-0c6c-4415-adb0-b8393b262f15.png)

Gibt es irgendwie eine Lösung oder habe ich irgendetwas falsch gemacht?

Vielen Dank im Voraus und viele Grüße

**Comment by murksmax66** [comment]
Moin,
ich weiß ja nicht, ob Du Dein Problem zwischenzeitlich (ist ja schon einige Zeit ins Land gegangen) hast lösen können ... Ich habe das bei mir folgendermaßen gemacht (CCU3/raspi-matic):

- Einstellungen -> Gruppen -> Neu
- Gruppe benennen und dann die Entsprechenden Geräte in die (Heiz-)Gruppe hinzufügen, sprich den Raumthermostaten, ggf. Fenstersensoren, FBH-Regler oder Heizkörperthermostaten. das funktioniert aber nur, wenn die Geräte beim Einfügen in die Gruppe keine  Direktverbindung haben! 
- Die Gruppe speichern, damit wird ein virt. Gerät für diesen "Heizkreis" angelegt.
- Einstellungen wie Heizprofile, Fensteröffnung etc. dann nur noch in der Gruppe einstellen, nicht mehr direkt an dem Gerät WTH-x oder Fenstersensor oder Raumthermostat!
- Dann in HAP den vorhandenen Raumthermostaten löschen und ein neues Gerät hinzufügen. Diesmal aber das neu angelegte virt. Gerät der Heizgruppe hinzufügen (Kanal 1).

![IMG_2660](https://github.com/user-attachments/assets/93c1660d-f093-47bd-8bf7-c70cfe401111)

Und schon hat man in @Home eine Anzeige, die beim Anklicken die Temperatur und die Luftfeuchtigkeit angibt :)
